### DISCUSSION TITLE
The network between the Jedis and a master node is disconnected in cluster mode.

### DISCUSSION THREAD

**Discussion by Ruanjq98** [xetorthio/jedis#3675] [Q&A]
background page:
The network between the Jedis cluster client and a master node is disconnected.

What I want to do:
Check whether the network between the Jedis client and all master nodes is connected and whether the cluster status is normal. Is there any elegant way to achieve it?

**Comment by sazzad16** [comment]
In JedisCluster:
- You can use `getClusterNodes()`, iterate to map, get Connection and do whatever you want on each Connection.
- You can call `ping()`. If everything is alright, you'll get `PONG`. Otherwise, you'll get reply/error of each node.

**Reply by Ruanjq98** [reply]
Consider providing an API? API used to determine whether the cluster is currently available from the perspective of the client?
